docs: add CODE_OF_CONDUCT.md

joshuajames-smith opened the pull request:
## Description

Large documentation addition of CODE_OF_CONDUCT.md which improves the gliff.ai GitHub Community Health Files. 

*(note: file name uses `_` instead of `-` to meet GitHub's requested format which also allows for the documentation to be recognised by the system)

joshuajames-smith committed:
docs: add CODE_OF_CONDUCT.md

Large documentation addition of CODE_OF_CONDUCT.md which improves the gliff.ai GitHub Community Health Files. *(note: file name uses `_` instead of `-` to meet GitHub's requested format which also allows for the documentation to be recognised by the system)

joshuajames-smith committed:
style: remove spacing

Remove double space styling for better whole text consistency.

joshuajames-smith committed:
fix: ToC links

Fix text in `Table of Contents` links so navigation works.

joshuajames-smith committed:
style: edit typo

Fix spelling error in ToC link.

joshuajames-smith committed:
feat: add links to gliff.ai website

Add links to any mention of `gliff.ai` in the introductions body text to offer link to website.

joshuajames-smith committed:
fix: change contact email

Change contact email.

joshuajames-smith committed:
docs: update email

Missed old contact email, replaced with new one.

ChasNelson1990 reviewed (commented):
Should this more closely match our own internal code of conduct? Which is a lot more details - could be good, could be bad?

joshuajames-smith commented:
> Should this more closely match our own internal code of conduct? Which is a lot more details - could be good, could be bad?

@ChasNelson1990 think having a long complex document might off-put contributors, maybe just need to identify key points from ours and keeping it simplified - assuming the gliff.ai team are really abiding by the internal code of conduct, so this maybe more so contributors?

joshuajames-smith committed:
docs: update introduction section

Updated and expanded `introduction` section via additions from gliff.ai's internal Code of Conduct.

joshuajames-smith committed:
docs: update CODE_OF_CONDUCT.md

Intregated more internal gliff.ai code of conduct into gliff.ai's github code of conduct.

ChasNelson1990 commented on CODE_OF_CONDUCT.md:
```diff
@@ -2,7 +2,11 @@
-The purpose of this guide is to ensure all contributors and projects within the [gliff.ai](https://gliff.ai) space have a unified Code of Conduct to create a positive and welcoming environment for all.
+The purpose of this guide is to outline [gliff.ai](https://gliff.ai)'s expectations of individuals involved in [gliff.ai](https://gliff.ai) activities. While incidents of unaccpetable behaviour are relatively infrequent, this document serves as a guide to form a clear set of standards to maintain a safe and welcoming space that avoids/prevents behaviours that prevent this, while outlining reporti ...
+
+[gliff.ai](https://gliff.ai) employees a team of passionate and innovative backgrounds, with the outlook that our compnay, our products and our customers benefit from diveristy of backgrounds and perspectives. We prioritise creating a space which is a safe and welcoming environment for all, regardless of their personal identity or background, as anything other than thise can jeopardise the abilit ...
```
We employee a team of backgrounds do we?

joshuajames-smith replied:
Opps my bad! 🥴

joshuajames-smith committed:
chore: proof reading

Spell check the full document to ensure correct spelling. Add missing words to ensure document makes sense.
